src/text_processing_util_mds24/text_processing_util_mds24.py

```diff
@@ -5,6 +5,27 @@ def text_clean(docs: list[str]) -> list[str]:
 
 
 def frequency_vectorizer(docs: list[str]) -> list[dict[str:float]]:
+    """
+    Calculates the frequency of each word in a list of text documents.
+
+    Parameters
+    ----------
+    docs : list[str]
+        A list of text documents.
+
+    Returns
+    -------
+    list[dict[str: float]]
+        A list of dictionaries, where each dictionary contains the word and the word's frequency in a text document.
+        
+    Examples
+    --------
+    >>> documents = ["This is a sample document.", "Another document for testing."]
+    >>> result = frequency_vectorizer(documents)
+    >>> print(result)
+    [{'This': 0.14, 'is': 0.14, 'a': 0.14, 'sample': 0.14, 'document.': 0.14},
+    {'Another': 0.16, 'document': 0.16, 'for': 0.16, 'testing.': 0.16}]
```

Comment:
I think there is a calculation issue here. Please review. Thanks.

```suggestion
    [{'This': 0.2, 'is': 0.2, 'a': 0.2, 'sample': 0.2, 'document.': 0.2},
    {'Another': 0.25, 'document': 0.25, 'for': 0.25, 'testing.': 0.25}]
```